.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,10 @@
+contact_links:
+  - about: "Ask and answer user questions about Python on discuss.python.org."
+    name: "Getting help"
+    url: "https://discuss.python.org/c/users/7"
+  - about: "Large feature proposals, for example proposals for new syntax, should be discussed on an ideas forum rather than the issue tracker."
```

This line is a bit verbose.  Maybe something like this:
```suggestion
  - about: "Submit major feature proposal (e.g. syntax changes) to Discourse first."
```

---

The python-ideas mailing list is still an acceptable forum to discuss speculative ideas, so I think "ideas forum" is probably better than Discourse?

```suggestion
  - about: "Submit major feature proposal (e.g. syntax changes) to an ideas forum first."
```